test/config.test.ts

```diff
@@ -85,6 +86,7 @@ test("Config Override By Id", () => {
   template.hasResourceProperties("AWS::SQS::Queue", {
     KmsMasterKeyId: "alias/aws/sqs",
   });
+  template.hasResourceProperties("AWS::SQS::Queue", {});
```

Comment:
You need to have an explicit check that the property is absent (see [here](https://github.com/awslabs/aws-ddk/blob/278-configurator-doesnt-work-as-expected-when-using-resource-ids/test/config.test.ts#L84))